docs/source/asr/configs.rst

```diff
@@ -256,3 +291,104 @@ For example, a decoder config corresponding to the encoder above would look like
       feat_in: *enc_filters
       vocabulary: *labels
       num_classes: 28   # Length of the vocabulary list
+
+Citrinet
+~~~~~~~~
+
+The `Citrinet <./models.html#Citrinet>`__ and `QuartzNet <./models.html#QuartzNet>`__ models are very similar, and as such the components in their configs are very similar as well. Citrinet utilizes Squeeze and Excitation, as well as sub-word tokenization, in contrast to QuartzNet.
+
+Both architectures use the ``ConvASREncoder`` for the ``encoder``, with parameters detailed in the table above.
```

Comment:
How about adding some cons of Citrinet comparing to QuartzNet? about larger context? Speed-wise? not a number but mention due to the downsampling...